app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/HedvigDateTimeFormatter.kt

```diff
@@ -12,6 +12,14 @@ fun hedvigDateTimeFormatter(locale: Locale): DateTimeFormatter {
   return DateTimeFormatter.ofPattern("d MMM yyyy", locale)
 }
 
+/**
+ * Example output: "16 Jan"
+ */
+fun hedvigMonthDateTimeFormatter(locale: Locale): DateTimeFormatter {
```

Comment:
This is not a blocker for this PR, but just writing it here so we take care of it eventually.

In the chat feature, I also needed like a bunch of different formatters, so I did this https://github.com/HedvigInsurance/android/blob/7c5bb59b2ff7128ab7c8a50ace63c254d2679dd4/app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/HedvigDateTimeFormatter.kt#L44-L45 to make it easier to namespace them into `HedvigDateTimeFormatterDefaults`.
Taking inspiration from https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/commonMain/kotlin/androidx/compose/material/Button.kt;l=293?q=ButtonDefaults as they do it in the material library.

We can probably after everything is merged and we have the time try to consolidate all of our date formatting usages inside there, to make it easy to reason about what formatting options we're using overall.

No action needs to be taken in this PR, that code was only merged with the chat feature. Just to keep in mind for the future.